pipeline/secret_test.go

```diff
@@ -536,7 +566,7 @@ func testSecrets() *SecretSlice {
 				Image:       "vault:latest",
 				Name:        "vault",
 				Number:      1,
-				Pull:        "always",
+				Pull:        "build_start",
```

Comment:
These secret plugin blocks will still be `pull: always`, as that is referring to the vault image